include/unk_0203E348.h

```diff
@@ -14,10 +14,16 @@
 struct UseMailWork;
 struct ApricornBoxWork;
 struct PalPadWork;
-struct BerryPotsWork;
 struct UnownReportWork;
 struct GracideaWork;
 
+typedef struct BerryPotsWork {
```

Comment:
We're trying to phase out 'work' terminology, ``BerryPots`` or ``BerryPotsData`` would work just fine